pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -1158,6 +1140,31 @@ func handleModify(ctx *verifierContext, config *types.VerifyImageConfig,
 			status.Safename)
 	}
 
+	if !status.IsContainer && status.State == types.DOWNLOADED {
+		// Modify request, but status is in DOWNLOADED state. This happens
+		// during reboot. Compute the SHA
+		_, _, verifiedFilename := status.ImageDownloadFilenames()
+		imageHash, err := computeShaFile(verifiedFilename)
```

Comment:
This should probably do the signature check as well, but the old code didn't so that so its OK.